AUTHORS

```diff
@@ -1667,5 +1665,231 @@ George Bezerra <georgelione@gmail.com>
+Minya Liang <mliang@usaalabs.com>
+Kinnan Kwok <kkwoker@gmail.com>
+Adil L <adilio@gmail.com>
+Seth Holladay <me@seth-holladay.com>
+Chris Andrews <cpandrews8@gmail.com>
+Matt Reed <matthewreed26@gmail.com>
+Joe Henry <johenry@me.com>
+Alireza Alidousti <alireza.eva.u23@gmail.com>
+James Beavers <jamesjbeavers@gmail.com>
+Cameron Burwell <cburwell@users.noreply.github.com>
+Jakub Mrowiec - Alkagar <alkagar@gmail.com>
+Oliver Luebeck <oliver_luebeck@hotmail.com>
+Chris Jimenez <jimeneechris123@gmail.com>
+James Hodgskiss <james.hodgskiss@bazaarvoice.com>
+Guilherme Akio Sakae <akio.xd@gmail.com>
+Martin Michaelis <code@mgjm.de>
+Christopher Sidebottom <chris@damouse.co.uk>
+Edward Andrew Robinson <earobinson@gmail.com>
+Nigel Kibodeaux <nigelmail@gmail.com>
+Shakeel Mohamed <contact@shakeel.xyz>
+Tobias Kieslich <tobias.kieslich@gmail.com>
+Ruy Adorno <ruyadorno@hotmail.com>
+Stefania Sharp <stefaniasharp@gmail.com>
+Pawel Golda <pawelgolda@gmail.com>
+Steven Scott <Contact@TheDgtl.net>
+Alex Gilbert <alex@punkave.com>
+Siyuan Gao <siyuangao@gmail.com>
+Nicola Del Gobbo <nicoladelgobbo@gmail.com>
+Josh Lim <joshualim92@icloud.com>
+Feon Sua <feonss@gmail.com>
+Shawn McGinty <shawn@codouken.com>
+Jason Walton <dev@lucid.thedreaming.org>
+Jonathan Gourlay <gourlayjd@linux.com>
+Peter Paugh <ppaugh@chariotsolutions.com>
+Gregor Martynus <gregor@martynus.net>
+Joel Dart <jdart@remember.com>
+Tri Nguyen <tri@tridnguyen.com>
+Kasim Doctor <kasim.doctor@appdirect.com>
+Steve Jenkins <steve@skysteve.com>
+AlexeyM <alexey@malyshev.info>
+Nicolas Chaulet <n.chaulet@gmail.com>
+Adarsh Honawad <adarsh2397@gmail.com>
+Tim Ermilov <yamalight@gmail.com>
+ekulnivek <ekulnivek@hotmail.com>
+Ethan Brown <e@zepln.com>
+Lewis Zhang <lewiszhang@gmail.com>
+Kat Rosario <katrinajoy.rosario@fastretailing.com>
+jpaulptr <jpaulptr@gmail.com>
+Donovan Buck <donovan@donovan.bz>
+Toby Farley <tobyfarley@gmail.com>
+Suresh Srinivas <suresh.srinivas@intel.com>
+Alberto Lopez de Lara <albertolopezdelara@gmail.com>
+Jem Bezooyen <jem@sendwithus.com>
+Bob Clewell <bob@punkave.com>
+Raj Parekh <raj.parekh@aa.com>
+Tom Boutell <tom@punkave.com>
+Cristian Peñarrieta <cristian.penarrieta@gmail.com>
+Christian Murphy <Christian.Murphy.42@gmail.com>
+Dolapo Toki <dolapotoki@gmail.com>
+Shaopeng Zhang <szhang351@bloomberg.net>
+Matthew Meyer <mameye@microsoft.com>
+Chad Zezula <czezula@silasg.com>
+Eric Freiberg <eric.freiberg@pointsource.com>
+Mabry Cervin <Mabry.Cervin@inin.com>
+shaohui.liu2000@gmail.com <shaohui.liu2000@gmail.com>
+Chi-chi Wang <chichi@wangchiyi.com>
+Roger Jiang <rogerj.2010@gmail.com>
+Cheyenne Arrowsmith <hello@alwayscheyenne.com>
+Tim Chon <devchon@gmail.com>
+Michael Pal <michael.pal@ubc.ca>
+Fadi Asfour <fadi@wordy.com>
+Christina Chan <christina.chan@sap.com>
+Alessandro Vergani <alessandro.vergani@gmail.com>
+Ali Groening <ali.groening@gmail.com>
+Mike Fleming <flemingatx@gmail.com>
+WeiPlanet <xwjiang@gmail.com>
+243083df <243083df@dispostable.com>
```

Comment:
Hmm this account can be stolen via the publicly visible disposable email address.

Reply:
what can we do?

Reply:
Not sure, remove? edit?

Reply:
I if just remove it, it will appear again next time.
I can add it to `.mailmap` as "Anonymous".